core/src/main/resources/hudson/tasks/ArtifactArchiver/help-artifacts.html

```diff
@@ -1,7 +1,8 @@
 <div>
    You can use wildcards like 'module/dist/**/*.zip'.
    See <a href='http://ant.apache.org/manual/Types/fileset.html'>
-   the includes attribute of Ant fileset</a> for the exact format.
+   the includes attribute of Ant fileset</a> for the exact format
+   (except that space is not supported as a separator).
```

Comment:
I think there should be some more call-out to "space" the character.
```suggestion
   -- except that <tt>" "</tt> (<em>space</em>) is not supported as a separator.
```